lib/OpenQA/BuildResults.pm

```diff
@@ -102,9 +103,19 @@ sub compute_build_results {
                 clone_id => undef,
             },
             {order_by => 'me.id DESC'});
-        my %jr = (oldest => DateTime->now, passed => 0, failed => 0, unfinished => 0, labeled => 0, softfailed => 0, skipped => 0, total => 0);
+        my %jr = (
+            oldest     => DateTime->now,
```

Comment:
Because the hashref below did fit into 120 columns when folded i imagine. Doesn't seem like an inconsistency to me.